src/main/java/duke/task/Task.java

```diff
@@ -0,0 +1,114 @@
+package duke.task;
+
+import java.io.Serializable;
+import java.time.format.DateTimeFormatter;
+
+import duke.DukeException;
+
+/**
+ * Duke helps to serve as a reminder/scheduler.
+ * The user's tasks are be keyed into Duke, and will be held by Duke.
+ * The task class represents the task objects that will be keyed into Duke.
+ *
+ * @author Dargo
+ */
+public class Task implements Serializable {
+
+    // task = Full input line
```

Comment:
What is the purpose of these comments? If required please move them to the corresponding lines so that it will more clear to the reader

Reply:
Removed